crates/credentials/src/verifiable_credential.rs

```diff
@@ -130,6 +130,52 @@ pub struct VerifiableCredential {
     pub credential_subject: CredentialSubject,
 }
 
+impl VerifiableCredential {
```

Comment:
What do you think of using the `From` trait here? It automatically implements the corresponding `into()` method.

```suggestion
impl From<VerifiableCredential> for JwtPayloadVerifiableCredential {
```

Reply:
oh thats pretty cool, yea I'm about this (less code, but less flexibility?) Seems like we dont need any special case stuff so I'll add it

Reply:
updated!